src/LeanCode.ContractsGenerator/Analyzers/ExternalTypeCheck.cs

```diff
@@ -6,11 +6,13 @@ public class ExternalTypeCheck : BaseAnalyzer
 {
     public const string Code = "CNTR0004";
 
-    private ImmutableHashSet<string> knownTypes = ImmutableHashSet<string>.Empty;
+    private ImmutableHashSet<string> knownTypes = ImmutableHashSet.Create(
+        "LeanCode.Contracts.IProduceNotification",
```

Comment:
Let's exclude both of them from ever being generated. It is redundant:
1. `INotification` does not give any meaning,
2. `IProduceNotification` is being translated to `Topic.notifications` field either way.